.github/workflows/main.yml

```
@@ -0,0 +1,32 @@
+name: Format Fortran Code with fprettify
+
+on: [push, pull_request]
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)